backend.native/tests/build.gradle

```diff
@@ -3739,7 +3739,7 @@ createInterop("embedStaticLibraries") {
     // Note: also hardcoded in def file.
     final String libDir = "$buildDir/embedStaticLibraries/"
 
-    it.getByTarget(target.name).doFirst {
+    it.getByTarget(target.name).get().doFirst {
```

Comment:
May be use `configure { ... }` instead of `get()`?

Reply:
Tnx, fixed